README.md

```diff
@@ -36,3 +36,153 @@ This is the shared library defining public classes that constitute the worker in
 ### worker-document-testing
 
 This contains implementations of the testing framework to allow for integration testing of the Document Worker. The project can be found in [worker-document-testing](worker-document-testing).
+
+## Workflow event handlers
+
+The below are the various event handlers from the workflow scripts.
```

Comment:
Try to make it a bit more clear that not every javascript file requires all of these functions, and that the event will only be triggered if there is a corresponding function within the javascript file being executed. Like for example, our workflow scripts do not contain a onBeforeProcessDocument function. Someone reading this might assume they need to implement them all even though they might not need to for their use case.